.gitlab/build/xliff-core-1.2-strict.xsd

```diff
@@ -0,0 +1,2223 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Isn't there some dependency we can use to import this file from? Otherwise, we'd need to maintain the XSD here for all eternity, and I'd like to avoid that.